src/main/kotlin/com/epam/ahnl/model/entity/Student.kt

```diff
@@ -0,0 +1,30 @@
+package com.epam.ahnl.model.entity
```

Comment:
I'll write here, but it's about all package names. I understand that you've just moved code from sub modules, but pls check  package names.
For ex. ```model.entity``` is redundant, use only 1 name

Reply:
The application was divided into three layers and folders, respectively: controller, service, repository. These folders contain subfolders.